tensorflow/stream_executor/cuda/cuda_activation.cc

```diff
@@ -27,8 +27,7 @@ CudaContext* ExtractCudaContext(CUDAExecutor *cuda_exec);
 CUDAExecutor *ExtractCudaExecutor(StreamExecutor *stream_exec);
 
 ScopedActivateExecutorContext::ScopedActivateExecutorContext(
-    CUDAExecutor *cuda_exec)
-    : cuda_exec_(cuda_exec),
+    CUDAExecutor *cuda_exec):
```

Comment:
@vrv I think this is suspicious, not sure who owns the stream_executor but could it be that because we are not deleting the cuda_exec once it scopes out, will cause #1727 ? I did not have time to dig in the code to find out .